python/mxnet/contrib/text/embedding.py

```diff
@@ -610,25 +591,6 @@ class FastText(_TokenEmbedding):
         embedding vectors, such as loaded from a pre-trained token embedding file. If None, all the
         tokens from the loaded embedding vectors, such as loaded from a pre-trained token embedding
         file, will be indexed.
-
-
-    Properties
-    ----------
-    token_to_idx : dict mapping str to int
-        A dict mapping each token to its index integer.
-    idx_to_token : list of strs
-        A list of indexed tokens where the list indices and the token indices are aligned.
-    unknown_token : hashable object
-        The representation for any unknown token. In other words, any unknown token will be indexed
-        as the same representation.
-    reserved_tokens : list of strs or None
```

Comment:
I don't see this being added back elsewhere...

Reply:
Those doesn't really exists in the class. They are copy pasted from parent class.